go/engine/kex2_provisionee.go

```diff
@@ -288,6 +290,21 @@ func (e *Kex2Provisionee) handleDidCounterSign(sig []byte, sdhBoxes []keybase1.S
 		return err
 	}
 
+	// remember how to update a user with the new links
+	e.localDelegateFn = func(u *libkb.User) (err error) {
```

Comment:
Do you mean the signing key signed the wrong chainlink when adding the paper key?